app/views/content/train-to-be-a-teacher/promos/_find-your-ske-course.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="find-your-course">
+  <div class="left">
+    <div class="icon">
+       <%= image_pack_tag("media/images/icon-search-black.svg", alt: "") %>
+    </div>
+  </div>
+
+  <div class="right">
+    <h2 class="heading-m heading--margin-top-0">Find your SKE course</h2>
+    <p>
+      Take a look at the different SKE courses available.
```

Comment:
should we say `subject knowledge enhancement` in full in either the heading or body @EllieNodder ?